content/docs/getting-started/ocm-cli/download-resources-from-component-versions.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Download Resources from Component Versions"
```

Comment:
The _Next_ here is `Example of a Component Descriptor`. But, that next points to Deploying a Helm Chart which shouldn't be the next. The next should be introduction in the ocm-controller section.